src/main/resources/expected.csv

```diff
@@ -0,0 +1,7 @@
+fruit,quantity
+banana,100
+orange,100
+kiwi,150
+blackberries,400
+apricots,200
+mango,300
```

Comment:
I do not know why, but in the end of every csv (or any other) file is needed to be empty line